src/Exceptionless.Core/Extensions/MailerExtensions.cs

```diff
@@ -29,7 +30,27 @@ public static class MailerExtensions {
             return notification;
         }
 
-        public static System.Net.Mail.MailMessage ToMailMessage(this MailMessage notification) {
+        public static MimeMessage ToMailMessage(this MailMessage notification) {
+            var message = new MimeMessage();
+            var builder = new BodyBuilder();
+
+            if (!String.IsNullOrEmpty(notification.From))
```

Comment:
I found a bug yesterday and fixe in the old implementation, where the web.config settings can't be picked up from azure functions (and has been removed now..) We need to add an else clause here kinda like this to fix this issue:

```
            if (!String.IsNullOrEmpty(notification.From))
                message.From = new MailAddress(notification.From);
            else
                message.From = new MailAddress(Settings.Current.SmtpFrom);
```